userguide/one_shot_object_detection/README.md

```diff
@@ -44,81 +44,67 @@ model.save('mymodel.model')
 model.export_coreml('MyCustomOneShotDetector.mlmodel')
 ```
 
-Previewing the synthetic images generated for training is useful to gauge the kind of test images the model would be trained to handle. We can preview the augmented images generated and used by the model for training as follows:  
-
-
-```python
-augmented_images = \
-	tc.one_shot_object_detector.util.preview_synthetic_training_data(starter_images, 'label')
-
-augmented_images.explore()
-```
-
-![Explore augmented images](images/augmented_images_explore.jpg)
-
-Some examples of synthetic training data generated:
-
-![Sample augmented images](images/augmented_images_collage.jpg)
-
-*Note: In order to view the exact augmented images used to train the model, the same seed should be used which was used to train the model.*
 
-
-To visualize the predictions made on the test set:
-
-
-```python
-test_data['image_with_predictions'] = \
-	tc.object_detector.util.draw_bounding_boxes(test_data['image'], test_data['predictions'])
-
-test_data.explore()
-```
+Visualizing predictions made on the test set:
 
 ![Prediction test data](images/test_data_explore.jpg)
 
+
 Examples of test image predictions:
 
 ![Sample prediction image - 1 and 2](images/sample_prediction_images_1_2.jpg)
 
 
 #### How it works
 
-To begin training for one-shot object detector, a set of synthetic training data needs to be generated using the provided starter training image(s). These images are augmented by applying yaw, pitch, and roll rotations. These operations are applied on a set of background images. The background images may be provided by the user. If no background images are provided, a set of default background images are automatically downloaded and used.
+In this section, we will go into detail on what happens under the hood of the `create` method. An OSOD model is trained in two steps: First, we use the starter images to generate synthetic data and annotations, and second, we train an object detector.
 
-To learn more about the training implementation details, refer to the [Object Detector: How it Works](https://apple.github.io/turicreate/docs/userguide/object_detection/how-it-works.html) chapter.
+To begin training, a set of synthetic training data needs to be generated using the provided starter image(s). 
+The synthetic images are generated by applying yaw, pitch, and roll rotations on a set of background images. The background images may be provided by the user. If no background images are provided, a set of default background images are automatically downloaded and used.
 
+You can preview the synthetic images the toolkit generates by calling the `preview_synthetic_training_data` utility. This helps you understand data the model would be trained on:  
 
-#### Deployment to CoreML
 
-To learn more about deploying One-Shot to CoreML details, refer to the [Object Detector: Deployment to Core ML](https://apple.github.io/turicreate/docs/userguide/object_detection/export-coreml.html) chapter.
+```python
```

Comment:
On reading this, I am more convinced that we should have an advanced section that focusses only on the backgrounds and a place to add more things in the future. The first exposure (intro, how it works) should be very simple.  The how-it-works section seems to go back and forth between explaining how it works and explaining how to use more advanced options.